.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,12 @@
 
 #### What does this implement or fix?
 
+## Change Type (Required)
```

Comment:
Can we just enforce these with labels on the PR rather than these checkboxes?

Reply:
It is possible yes, maybe even clearer. I will enforce it so we can't merge without the labels. Also I think we can use the same set of labels for releases and release notes, so that we don't have e.g. both `minor` and `feature` but just the one that would be used for both release versions and release notes